jenkinsfiles/ginkgo-kernel.Jenkinsfile

```diff
@@ -37,6 +37,10 @@ pipeline {
             returnStdout: true,
             script: 'if [ "${JobKernelVersion}" = "net-next" ]; then echo -n "0"; else echo -n ""; fi'
             )}"""
+        RUN_QUARANTINED="""${sh(
```

Comment:
Don't we need to update other Jenkinsfile as well? Also (unrelated to your PR), but can we somehow share some sections among all Jenkinsfiles (e.g. ENV vars)?

Reply:
@brb this is only for running https://jenkins.cilium.io/view/Quarantine%20Pipelines/ which for now only use this jenkinsfile.